java-dates/src/test/java/com/baeldung/timestamp/TimestampToStringConverterTest.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.timestamp;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Timestamp;
+
+class TimestampToStringConverterTest {
+
+    @Test
+    public void testConversion() {
```

Comment:
I mentioned this in my article review, but just to place it here, too, make sure that the method names adhere to the site standard of givenXXX_whenYYY_thenZZZ or whenYYY_thenZZZ